ansible/roles/test/vars/testcases.yml

```diff
@@ -192,3 +192,10 @@ testcases:
     crm:
       filename: crm.yml
       topologies: [t1, t1-lag, t0, t0-56, t0-64, t0-116]
+
+    dip_sip:
+      filename: dip_sip.yml
+      topologies: [t0, t1, t1-lag]
```

Comment:
We need t0-56 at least. It would be better to add all.

Reply:
@qiluo-msft We need to discuss which topologies must be supported by this test case. Regarding t0-56, I have no way to check it. Basically, we can use any topology with this testcase which contains at least two PORT/LAG RIFs connected to neighbors, providing the valid LLDP info.

Reply:
Then it's ok to add all known topo here. Others can help you test.

Reply:
@qiluo-msft I have extended this testcase with the additional topologies as we discussed.